torch/distributed/nn/api/remote_module.py

```diff
@@ -322,6 +346,7 @@ class RemoteModule(_RemoteModule):
             >>> module_cls = MyModule
         args (Sequence, optional): args to be passed to ``module_cls``.
         kwargs (Dict, optional): kwargs to be passed to ``module_cls``.
+        device (str): Device on the destination worker where we‘d like to place this module.
```

Comment:
Consistent with rest of PyTorch (ex: .to()) docs we should make this `device (torch.device):` ....

Reply:
Reverted the previous change from torch.device to str.

Reply:
Changed it back to torch.device.